admin_manual/maintenance/migrating_owncloud.rst

```diff
@@ -34,8 +36,21 @@ See the table below for a version map, where migrating is easily possible:
           so we can at least perform some basic migration tests, before you
           migrate your production instance.
 
-After downloading the correct version of Nextcloud from our
-`older releases page <https://nextcloud.com/changelog/>`_,
-proceed like described in the :doc:`manual_upgrade` manual.
+1. First download the correct version of Nextcloud from our `older releases page <https://nextcloud.com/changelog/>`_,
+
+2. Make sure to have do a :doc:`backup<backup>` before migrating.
+
+3. Follow the upgrade instructions described in the :doc:`manual_upgrade` manual.
+
+4. When migrating to Nextcloud 20.0 or later, you will also need to run the following commands after ``occ upgrade``:
```

Comment:
optionally link to the occ page for those who don't know how to run occ

Reply:
If they have followed the previous step and went to "manual_upgrade" they should already know.